tests/test_rooms/test_query_rooms.py

```diff
@@ -78,6 +80,17 @@ def test_calendar_list_function(self, mocked_method):
         get_google_api_calendar_list()
         assert mocked_method.called
 
+    def test_all_remote_rooms_with_true(self):
```

Comment:
@MCFrank16 please change the grammar in the function name. `test_all_remote_rooms_with_true` makes no sense. Perhaps you could try `test_all_remote_rooms_with_returnall_true`.